checks/fileparser/github_workflow.go

```diff
@@ -298,6 +298,10 @@ func IsWorkflowFile(pathfn string) bool {
 	}
 }
 
+func IsWorkflowFileCb(filename string) (bool, error) {
```

Comment:
Why have a `func` that never returns error as part of the return?

Reply:
Could we remove the `error`?

One line `go` funcs are discouraged. Is there a specific reason to have this in a separate func?